src/views/Main/Catalog/Catalog.js

```diff
@@ -1,7 +1,7 @@
-import React, { PropTypes as T } from 'react'
-import {Button} from 'react-bootstrap'
-import AuthService from 'utils/AuthService'
-import styles from './styles.module.css'
+import React, { PropTypes as T } from 'react';
```

Comment:
what is this for `{ PropTypes as T }`? doesn't look like `PropTypes` is being used, nor `T`